app/components/accountpage.jsx

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {Link} from 'react-router';
+
+class AccountPage extends Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            loggedIn: false,
```

Comment:
looks like we may not need to have state here? if we do, we could set loggedIn to this.props.state.auth